templates/my.cnf.j2

```diff
@@ -1,3 +1,7 @@
+
+# Don't edit manually!
+# {{ ansible_managed }}
```

Comment:
The `{{ ansible_managed }}` line should suffice. Can you change the files to just have two new lines—first:

```
# {{ ansible_managed }}
```

Then a blank newline before the start of the rest of the file? That follows the convention I use in many of my other roles (e.g. https://github.com/geerlingguy/ansible-role-redis/blob/master/templates/redis.conf.j2 or https://github.com/geerlingguy/ansible-role-ntp/blob/master/templates/ntp.conf.j2).